Makefile

```diff
@@ -96,7 +96,3 @@ tests:
 .PHONY: tests-notebooks
 tests-notebooks:
 	pytest --nbmake "notebooks/" --nbmake-timeout=600 --ignore=notebooks/dask_from_CLI.ipynb
-
-.PHONY: tests-all
-tests-all:
-	pytest
```

Comment:
@wigging Why was this removed?

Reply:
I saw it was duplicate. Thanks

Reply:
Yes, it is a duplicate to the other test command. Don't need two commands in the make file that do the same thing. Again, if you would make these changes as pull requests, then review the pull requests, mistakes like this can be avoided.